DataDog/storedog-backend
# Create new ads service in Java

## Description

This PR adds a new service called `ads-java`. It is basically a duplicate of the existing `ads` service except it's written in Java. The purpose here is to create a Java service so the container can be used to display Java profiling or any other DD product with a Java integration.

## How to test
1. Pull down this branch, run `docker-compose up`
2. Go to frontend repo
3. In `env.local`, switch the `NEXT_PUBLIC_ADS_PORT` to be 3030
4. Start frontend
5. Make sure ads continue to be served on the homepage


Before you move on, make sure that:
